core/src/test/java/com/graphhopper/util/GHUtilityTest.java

```diff
@@ -67,6 +68,16 @@ Graph initUnsorted(Graph g) {
         return g;
     }
 
+    double getLengthOfAllEdges(Graph graph) {
+        double distance = 0;
+        DistanceCalc calc = new DistanceCalc2D();
+        AllEdgesIterator iter = graph.getAllEdges();
+        while (iter.next()) {
+            distance += iter.fetchWayGeometry(3).calcDistance(calc);
```

Comment:
Is this also possible with iter.getDistance()?

Reply:
This does not work then. Will look a bit deeper whats wrong here.

Reply:
getDistance doesn't seem to verify this failure in the tests so kept as-is and added a comment.